cms/djangoapps/contentstore/views/videos.py

```diff
@@ -314,6 +315,7 @@ def videos_post(course, request):
             ("course_video_upload_token", course_video_upload_token),
             ("client_video_id", file_name),
             ("course_key", unicode(course.id)),
+            ("course_title", urllib.quote_plus(course.display_name.encode("utf-8"))),
```

Comment:
Could you please write a comment that this works, because the `edraak-video-processor` knows this and works fine with it.